src/docs/asciidoc/developer-manual/develop-cloud-services/initialization-code.adoc

```diff
@@ -0,0 +1,27 @@
+:relative-path: ../../
+include::{docdir}/variables.adoc[]
+
+ZetaPush provides you the possibility to add some initialization code in your _custom cloud service_.
+For this you need to implement the method `onApplicationBootstrap()` in your _custom cloud service_.
+
+.Initialization code
+[source, javascript, linenums]
+----
+import { Bootstrappable } from '@zetapush/core';
+
+export default class implements Bootstrappable {
+  async onApplicationBootstrap() {
+    console.log('Initialization');
+  }
+}
+----
+
+The implementation of the `Bootstrappable` interface is optional but it is a best practice, you just need to implement a method named `onApplicationBootstrap()`.
+Indeed, implementing it avoid you to write wrong implementation of `onApplicationBootstrap()` method.
```

Comment:
Indeed, implementing it avoid**s** you to write wrong implementation of `onApplicationBootstrap()` method. It also indicates to other developers that you really wanted to add initialization code to your _cloud service_ on purpose and it also provides documentation about onApplicationBootstrap method directly in code.